core/domain/user_domain.py

```diff
@@ -345,3 +348,23 @@ def validate(self):
             raise utils.ValidationError(
                 'Expected can_review_questions to be a boolean value, '
                 'found: %s' % type(self.can_review_questions))
+
+
+class ModifiableUserData(python_utils.OBJECT):
```

Comment:
Sorry, forgot an important word in the naming. Should be ModifiableProfileUserData.

Reply:
Actually we will also use this domain object to pass in the information for updating full users also on Android.
See Line 233 in `user_services_test`. In that case I think current name seems better to me. Wdyt?

Reply:
Or ModifiableProfileData, if you like (it's a bit shorter).

Reply:
Oh, sorry, just saw your comment. Good point. Yes, OK, the current name is fine, please disregard previous comments. But give more info in the docstring about all this context.

(In general, if a reviewer asks a question, something isn't clear and it's often worth updating / adding comments to the code as part of your response.)

Reply:
Are you sure it would be understandable that both profile and full users can pass information into this and not just "profile"? Just double checking with you on this.

Reply:
Sorry -- our comments are crossing each other. See [this one](https://github.com/oppia/oppia/pull/10241#discussion_r470875269). In short, keep the name you have now, but update the docstring (it was not clear to me that this was for all types of users).

Reply:
Reply to [this](https://github.com/oppia/oppia/pull/10241#discussion_r470875269) Sure, thanks for the advice! 
(Sorry sometimes new comments get showed up automatically and sometimes they don't due to internet)
I have added better explanation in the docstring now. Also I have renamed the class as `ModifiableUserDataV1` and referring to it as `modifiable_user_data` in the variables --this kind of naming looks fine?

Reply:
Yup, I think it's fine, thanks!